### DISCUSSION TITLE
Datadog API Host - Docs Update

### DISCUSSION THREAD

**Discussion by cjsaurusrex** [artilleryio/artillery#1675] [Artillery]
Hey all!

Sharing this here as the Artillery don't appear to be available for public contribution - so hoping someone here has access and can update!

**Context:**
We were having an issue using the `artillery-plugin-publish-metrics` plugin as our Datadog is within the EU region, not the US region and therefore the API endpoint is `api.datadoghq.eu`. It took quite a bit of searching, however we found that a fix for this was put in place [here](https://github.com/artilleryio/artillery-plugin-publish-metrics/commit/b70520c75122fb2497d7857304b3f71b25339fcb).

This adds a new configuration key of `apiHost`, and the Artillery docs don't document this so it caused quite a bit of confusion.

**Request(?)**
Can we get the [Datadog section of the Artillery docs](https://www.artillery.io/docs/guides/plugins/plugin-publish-metrics#datadog) updated to contain this new config key? (And potentially also specify that a `https://` prefix is not required and will break it)

**Comment by hassy** [comment]
docs updated, thank you @cjsaurusrex!

https://docs.art/guides/plugins/plugin-publish-metrics#datadog
